.github/ISSUE_TEMPLATE/community_call.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Community call plan
+about: Creating a Community call Agenda
+title: 'Community Call <Date>'
+labels: "kind/community_call"
+assignees: ''
+---
+  
+  ## Description
+   - [ ] Create Agenda 
+   - [ ] Group - Flatcar Container Linux User   
+   - [ ] Twitter 
+   - [ ] Add Youtube live stream / recording link
+
+## Links for participants:
+- Zoom link: https://us06web.zoom.us/j/82054240491
+- Meeting ID: 820 5424 0491
+- Passcode: 444888
+- Youtube live stream / recording: <link>
+- Release Planning Board: https://github.com/orgs/flatcar-linux/projects/5
+  
+  
+## Welcome
+- Brief introduction of new participants or attendees
+  
+## //Topics//
+  - <CAPI>
+  - <ARM64>
+  - <Security>
+  - <Tips>
+
+## Q&A
+- Questions from community participants, answered by the Flatcar maintainers.
```

Comment:
I would not mix up the checklist for Community meetings (i.e. the new issue type we're defining here) with the Agenda.